snare.conf.example

```diff
@@ -5,10 +5,13 @@
 listen = "<address>:<port>";
 
 github {
-  // The path to the directory where your per-repo programs are stored.
-  reposdir = "<path>";
-
   match ".*" {
+    // For each webhook coming in, this will execute the command:
```

Comment:
I wonder, can we add a test which checks the example config file is valid perhaps?

Reply:
I had to tweak things a bit but this is a good idea: fixed in https://github.com/softdevteam/snare/pull/53/commits/13d9afbe307501a0bd43bcebbbd6e06fc16c77d5.